Tests/OnlineRetailPortal.Tests/ProductCoreTestClass.cs

```diff
@@ -0,0 +1,65 @@
+﻿using OnlineRetailPortal.Contracts;
+using OnlineRetailPortal.Core;
+using OnlineRetailPortal.Mock;
+using OnlineRetailPortal.Mock.Models;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Xunit;
+
+namespace OnlineRetailPortal.Tests
+{
+    public class ProductCoreTestClass
+    {
+        [Fact]
+        public async void Test_to_check_aad_product_function_of_core()
+        {
```

Comment:
test name should be AddProduct_With_Valid_Request_Should_Be_Added_Successfully

Reply:
done